src/nvim/fileio.c

```diff
@@ -1992,13 +1992,15 @@ readfile (
 }
 
 #ifdef OPEN_CHR_FILES
-/*
- * Returns TRUE if the file name argument is of the form "/dev/fd/\d\+",
- * which is the name of files used for process substitution output by
- * some shells on some operating systems, e.g., bash on SunOS.
- * Do not accept "/dev/fd/[012]", opening these may hang Vim.
- */
-static int is_dev_fd_file(char_u *fname)
+/// Returns true if the file name argument is of the form "/dev/fd/\d\+",
+/// which is the name of files used for process substitution output by
+/// some shells on some operating systems, e.g., bash on SunOS.
+/// Do not accept "/dev/fd/[012]", opening these may hang Vim.
+///
+/// @param fname The file name to check
+///
+/// @return true if the file name argument is of the form "/dev/fd/\d\+",
+static bool is_dev_fd_file(char_u *fname)
```

Comment:
I can take them if needed. How can I do it?

On Wed, Jan 06, 2016 at 6:59 pm, Justin M. Keyes < notifications@github.com [notifications@github.com] > wrote:
In src/nvim/fileio.c
[https://github.com/neovim/neovim/pull/3698#discussion_r49026859] :

> -/*
> - * Returns TRUE if the file name argument is of the form "/dev/fd/\d\+",
> - * which is the name of files used for process substitution output by
> - * some shells on some operating systems, e.g., bash on SunOS.
> - * Do not accept "/dev/fd/[012]", opening these may hang Vim.
> - */
> -static int is_dev_fd_file(char_u *fname)
> +/// Returns true if the file name argument is of the form "/dev/fd/\d\+",
> +/// which is the name of files used for process substitution output by
> +/// some shells on some operating systems, e.g., bash on SunOS.
> +/// Do not accept "/dev/fd/[012]", opening these may hang Vim.
> +///
> +/// @param fname The file name to check
> +///
> +/// @return true if the file name argument is of the form "/dev/fd/\d\+",
> +static bool is_dev_fd_file(char_u *fname)


As part of lint? That would be good. The PRs by @war1025 [https://github.com/war1025] may need to be adopted by someone though.

—
Reply to this email directly or view it on GitHub [https://github.com/neovim/neovim/pull/3698/files#r49026859] .[https://github.com/notifications/beacon/AFXYhUnuMHTWK9eIXl4uP8L9fhbjkK4zks5pXaHIgaJpZM4GkTBh.gif]